mix.exs

```diff
@@ -20,7 +20,7 @@ defmodule PhoenixSlime.Mixfile do
   end
 
   def deps do
-    [{:phoenix, "~> 1.2"},
+    [{:phoenix, "~> 1.3-rc"},
```

Comment:
I'll test this out when I get a chance a bit later today, but not sure it's going to work, because the code changes require that `attrs/1` and `params/1` be under `Mix.Phoenix.Schema` and not `Mix.Phoenix` anymore...